packages/dbadmin/src/controllers/DBAdminApi.ts

```diff
@@ -55,7 +55,12 @@ export class DBAdminApiImpl implements DBAdminApi {
     createDatabase = async(request: e.Request, response: e.Response) => {
         logger.requestLog(` * createDatabase request received, with body of ${request.headers["content-length"]} bytes`)
         const result = await this.ctx.dbAdminApiWorker.createDatabase(CREATE_DATABASE_SQL)
-        lionwebResponse(response, result.status, {
+        // lionwebResponse(response, result.status, {
+        //     success: result.status === HttpSuccessCodes.Ok,
+        //     messages: [ {kind: "Info", message: result.queryResult} ]
+        // })
+        const result2 = await this.ctx.dbConnection.query({clientId: "Repository", repository: "public"}, removeNewlinesBetween$$(CREATE_GLOBALS_SQL))
```

Comment:
I would consider renaming this one to `createGlobalFunctionsResult`

Reply:
Indeed.

Reply:
Removed this variable as well, as there is no return value, only an exception if something goes wrong.